### PR TITLE
Update icons to new symbol icon api

### PR DESCRIPTION
**Pull request by DuMaM**
This PR add new svg icon from set used across whole Jenkins -> https://ionic.io/ionicons
This follow with more modernize Jenkins look and enable theming.
More here: https://weekly.ci.jenkins.io/design-library/Symbols/

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

Other connected PR's:
https://github.com/jenkinsci/workflow-job-plugin/pull/274
Note: This require latest LTS as a base: 2.346.1

After:
FireShot Capture 008 - qmk #181  Jenkins  - 192 168 0 31

### CODE REVIEW

**Review by jglick — COMMENTED** [review]
Has this been checked against the currently declared baseline version (2.332.1) as well as newer versions?

**Comment by DuMaM** [comment]
Ok I checked this and custom icons do not work on 2.332.

**Comment by jglick** [comment]
OK. Take out of draft when that is fixed, or if it seems not straightforward to fix and you decide to just update the baseline to the first LTS where the icons do work.

**Comment by DuMaM** [comment]
Symbol api was introduced in 2.346.1 there is no point of trying to fix this, bellow those versions. I bumped base line.

**Comment by DuMaM** [comment]
Here is PR connected to this one
https://github.com/jenkinsci/workflow-job-plugin/pull/274

**Comment by NotMyFault** [comment]
Guess I'll take a look here too.

**Review by NotMyFault — APPROVED** [review]
LGTM, redo symbol feedback is non blocking.

File: src/main/java/org/jenkinsci/plugins/workflow/cps/replay/ReplayAction.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ private ReplayAction(Run run) {
     }
 
     @Override public String getIconFileName() {
-        return isEnabled() || isRebuildEnabled() ? "redo.png" : null;
+        return isEnabled() || isRebuildEnabled() ? "symbol-redo plugin-workflow-cps" : null;
```
[the remark this answers is not in the document]
Comment (DuMaM) [reply src/main/java/org/jenkinsci/plugins/workflow/cps/replay/ReplayAction.java:105]: That was also my idea to place it there and make usable for everyone.

[the remark this answers is not in the document]
Comment (DuMaM) [reply src/main/java/org/jenkinsci/plugins/workflow/cps/replay/ReplayAction.java:105]: PR added for core.

**Review by jglick — APPROVED** [review]
Looks good, thanks!

**Comment by DuMaM** [comment]
I will add similar PR when my changes land on core.

**Pull request merged**